### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by prateek1361**

### CODE REVIEW

File: cart-page/cart-page.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
+.nav-brand{
+    font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    color: whitesmoke;
+    font-size: 30px;
+    }
```
Comment (Creator-singh) [inline cart-page/cart-page.css:5]: use rems instead of pixels

File: cart-page/cart-page.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        color:white;
+        font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    }
+    header{
+        position: fixed;
+        top: 0;
+        left: 0;
+        right: 0;
+        background:  #334366;
+       box-shadow: 0 8px 12px black;
```
Comment (Creator-singh) [inline cart-page/cart-page.css:41]: use variables for colors

File: cart-page/cart-page.css [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+      border-radius: .5rem;
+      border: 2px solid black;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      background-color: #334366;
+      box-shadow: 0 8px 12px black;
+    }
+    .navbar{
+        font-size: 20px;
```
Comment (Creator-singh) [inline cart-page/cart-page.css:18]: use rems

File: cart-page/cart-page.css [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    header{
+        position: fixed;
+        top: 0;
+        left: 0;
+        right: 0;
+        background:  #334366;
+       box-shadow: 0 8px 12px black;
+    }
+    form {
+        color: #555;
```
Comment (Creator-singh) [inline cart-page/cart-page.css:44]: use variables for colors

**Review by Creator-singh — CHANGES_REQUESTED** [review]
added few comments there are multiple instance apart from the comments where the variables and rems should have to be used please do the needful for code consistency.

File: cart-page/cart-page.css [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+    font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    color: whitesmoke;
+    font-size: 30px;
+    }
+    .nav{
+     padding: 5px 10px;
+      border: 2px solid ;
+      border-radius: .5rem;
+      border: 2px solid black;
+      display: flex;
```
Comment (SanketDhabarde) [inline cart-page/cart-page.css:11]: You can make utility class to center.

File: cart-page/cart-page.css [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+        padding: 2px;
+        border: 1px solid ;
+        border-radius: 5px;
+    
+      }
+      input[type="search"] {
+        border: none;
+        background:whitesmoke;
+        margin: 0;
+        padding: 7px 8px;
```
Comment (SanketDhabarde) [inline cart-page/cart-page.css:55]: Use utility class for padding, margins so that it will be consistent in your app. Also, use relative units.

**Review by SanketDhabarde — CHANGES_REQUESTED** [review]
Suggestions: properly format your code (use prettier), put comments in your CSS files, use relative units, use variables for color

**Pull request merged**